shared/wallets/wallet/settings/index.tsx

```diff
@@ -168,6 +197,46 @@ class AccountSettings extends React.Component<SettingsProps> {
               )}
             </Kb.Box2>
             <Divider />
+            {!!props.showExternalPartners && (
+              <Kb.Box>
+                <Kb.Box2
+                  direction="vertical"
+                  style={styles.section}
+                  fullWidth={true}
+                  gap="tiny"
+                  gapEnd={true}
+                >
+                  <Kb.Text type="BodySmallSemibold">External tools and partners</Kb.Text>
+                  {props.externalPartners.map((partner, index, array) => {
+                    const showDivider = index !== array.length - 1
+                    // Note: `props.user` is only the Keybase username if
+                    // this is the primary account. Non-primary accounts
+                    // are not associated with usernames.
+                    const url = partner.url
```

Comment:
maybe better for the connector to do this. then it can figure out the url and showDivider